src/vs/workbench/contrib/extensions/browser/extensionsViewlet.ts

```diff
@@ -396,8 +399,12 @@ export class ExtensionsViewPaneContainer extends ViewPaneContainer implements IE
 		addClass(parent, 'extensions-viewlet');
 		this.root = parent;
 
-		const header = append(this.root, $('.header'));
+		const overlay = append(this.root, $('.overlay'));
+		const overlayBackgroundColor = this.getColor(PANEL_DRAG_AND_DROP_BACKGROUND) ?? '';
```

Comment:
Please use `SIDE_BAR_DRAG_AND_DROP_BACKGROUND` as this section is side bar

Reply:
Fixed.